_targets.R

```diff
@@ -22,23 +24,19 @@ major_ion_names = c("Chloride","Sodium")
 # Define hydrologic event types in harmonized WQP data to exclude
 omit_wqp_events <- c("Spill","Volcanic action")
 
-list(
-  # Load harmonized WQP data product
-  tar_target(
-    p1_wqp_data,
-    readRDS(file = "1_fetch/in/DRB.WQdata.rds")
-  ),
-  # Filter harmonized WQP data for salinity data
-  tar_target(
-    p2_filtered_wqp_data,
-    filter_wqp_salinity_data(p1_wqp_data,major_ion_names,select_wqp_vars,omit_wqp_events)
-  ),
-  # Subset and save discrete specific conductance data
-  tar_target(
-    p2_wqp_spC_csv,
-    subset_wqp_spC_data(p2_filtered_wqp_data,fileout="2_process/out/DRB_WQdata_spC_data.csv")
-  ),
-  # Render data summary report
-  tarchetypes::tar_render(p3_wqp_spC_report, "3_visualize/src/report-wqp-salinity-data.Rmd",output_dir = "3_visualize/out")
-)
+# Define USGS specific conductance parameter codes
+SpC_pcodes <- c("00095","90095","00094","90096") 
+
+# Define minor HUCs (hydrologic unit codes) that make up the DRB (HUC04 = 0204)to use in calls to dataRetrieval functions
+drb_huc8s <- c("02040101","02040102","02040103","02040104","02040105","02040106",
+               "02040201","02040202","02040203","02040204","02040205","02040206","02040207",
+               "02040301","02040302")
+
+# Define USGS site types for which to download specific conductance data (include "Stream","Stream:Canal", and "Spring" sites)
+site_tp_select <- c("ST","ST-CA","SP") 
+
+
+# Return the complete list of targets
+c(p1_targets_list, p2_targets_list, p3_targets_list)
```

Comment:
I didn't notice this when I went through targets trainings, but these lists are unnamed. So, I'm not sure how to load the targets to inspect them. Did you find a way to do that? And should we name these lists with the target names?

Reply:
Yeah, I adopted the naming conventions from the targets trainings for these lists, but they could be made more informative, for example, by describing the phase (e.g. `p1_observations_targets_list`). I've kept the phase number naming convention for now, but let's revisit once we start adding modeling phases to the pipeline. 

I think these lists are more organizational than functional from a targets perspective. The lists keep the main `_targets.R` file cleaner by separating the targets out by phase, but to my knowledge you still need to call individual targets in order to inspect them using `tar_load` or `tar_read`. So I can't call `tar_load(p1_targets_list)` but I can call `tar_load(p1_wqp_data)` within the `p1_targets_list`.